downstream/modules/navigator/ref-faq-navigator.adoc

```diff
@@ -10,7 +10,7 @@ Where should the `ansible.cfg` file go when using an {ExecEnvNameSing}?:: The ea
 +
 Where should the `ansible.cfg` file go when not using an {ExecEnvNameSing} Ansible looks for the `ansible.cfg` in the typical locations when not using an {ExecEnvNameSing}. See https://docs.ansible.com/ansible/latest/reference_appendices/config.html[Ansible configuration settings] for details.
 
-Where should Ansible collections be placed when using an {ExecEnvNameSing}?:: The easiest place to have Ansible collections is in the project directory, in a playbook adjacent collections directory (for example, `ansible-galaxy collections install ansible.utils -p ./collections`).  The playbook directory is automatically mounted in the {ExecEnvNameSing} and {Navigator} finds the collections there. Another option is to build the collections into an {ExecEnvNameSing} using {Builder}. This helps content creators author playbooks that are production ready, since {ControllerName} supports playbook adjacent collection directories. If the collections are in another directory, set the `ANSIBLE_COLLECTIONS_PATHS` variable  and configure a custom volume mount for the directory. (See xref:ref-navigator-general-settings_settings-navigator[] for `execution-environment-volume-mounts`).
+Where should Ansible collections be placed when using an {ExecEnvNameSing}?:: The easiest place to have Ansible collections is in the project directory, in a playbook adjacent collections directory (for example, `ansible-galaxy collections install ansible.utils -p ./collections`).  The playbook directory is automatically mounted in the {ExecEnvNameSing} and {Navigator} finds the collections there. Another option is to build the collections into an {ExecEnvNameSing} using {Builder}. (See https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.1/html/ansible_builder_guide/assembly-intro-to-builder[Ansible Builder Guide] for more information on using {Builder}). This helps content creators author playbooks that are production ready, since {ControllerName} supports playbook adjacent collection directories. If the collections are in another directory, set the `ANSIBLE_COLLECTIONS_PATHS` variable  and configure a custom volume mount for the directory. (See xref:ref-navigator-general-settings_settings-navigator[] for `execution-environment-volume-mounts`).
```

Comment:
Some of the spaces after punctuation look like double spaces to me (though it may be my eyesight) 
for example,   'ansible
collections`).   The